### ISSUE TITLE
specplots not updating

### ISSUE THREAD

**Issue by prjemian** [APS-USAXS/ipython-usaxs#76]
... since August

**Comment by prjemian** [comment]
suspect the config file needs an update so it can search for data in November

**Comment by prjemian** [comment]
In `/home/beams/USAXS/Documents/eclipse/USAXS/livedata/buildSpecPlots.sh`, this is the present setting:

    SPEC_DATA_PATTERN=/share1/USAXS_data/2018-*/*.dat

That's not the problem now.

**Comment by prjemian** [comment]
The log file reports this problem:

```
#= 18330 --faulthandler---------- module enabled
[specplotsAllScans.py 18330 2018-11-12 10:40:02.293394] faulthandler: module enabled
Traceback (most recent call last):
  File "/home/beams/USAXS/Documents/eclipse/USAXS/livedata/specplotsAllScans.py", line 344, in <module>
    plotAllSpecFileScans(specFile)
  File "/home/beams/USAXS/Documents/eclipse/USAXS/livedata/specplotsAllScans.py", line 127, in plotAllSpecFileScans
    for scan_number in sd.getScanNumbers():
  File "/APSshare/anaconda/x86_64/lib/python2.7/site-packages/spec2nexus/spec.py", line 294, in getScanNumbers
    return sorted(self.scans.keys(), key=int)
ValueError: invalid literal for int() with base 10: '35.1'
#= 18317 --Done---------------------------------- Mon Nov 12 10:40:02 CST 2018
```

**Comment by prjemian** [comment]
That means the parser is still stuck on the file with multiple `#F` sections.

**Comment by prjemian** [comment]
Exception *should* also report the file name (absolute, full path)

**Comment by prjemian** [comment]
This needs repair in the [spec2nexus](https://github.com/prjemian/spec2nexus) repo in `spec2nexus/spec.py`

Could write a *temporary* wrapper to make that modification here until spec2nexus is updated.

**Comment by prjemian** [comment]
fix from APS-USAXS/livedata#3 works for me (dodge the problem and report it, then move on)
